lib/rubocop/cop/cop.rb

```diff
@@ -217,6 +217,25 @@ def excluded_file?(file)
         !relevant_file?(file)
       end
 
+      # This method should be overriden when a cop's behavior depends
+      # on state that lives outside of these locations:
+      #
+      #   (1) the file under inspection
+      #   (2) the cop's source code
+      #   (3) the config (eg a .rubocop.yml file)
+      #
+      # For example, some cops may want to look at at other parts of
```

Comment:
at ~at~

Reply:
Thanks! Fixed in https://github.com/rubocop-hq/rubocop/commit/e7fa9a916082e463ee803f979a5ceed55791c1c5